packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/SpanDetail/KeyValuesTable.test.js

```diff
@@ -57,107 +79,151 @@ describe('<KeyValuesTable>', () => {
     object: { a: 'b', x: 'y' },
```

Comment:
please replace with longer words to not accidentally match on letters from other words